docs/source/documentation/user-management.erb

```diff
@@ -0,0 +1,58 @@
+---
+title: User Management
+---
+
+## Adding users
+Use `./manage.py add-user` to add users
+
+```
+ Usage: manage.py add-user [OPTIONS] USERNAME
+
+╭─ Arguments ──────────────────────────────────────────────────────────────────────────────────────────────────╮
```

Comment:
This file doesn't seem to get correctly rendered when viewed on the web.

This is what it looks like when I build it locally:

![Screenshot 2024-12-18 at 13 42 51](https://github.com/user-attachments/assets/56cdcc06-a864-47ec-b923-4a9f2537f20b)